src/app/inventory/Stores.tsx

```diff
@@ -43,9 +44,13 @@ function Stores(this: void, { stores, buckets, isPhonePortrait }: Props) {
   const currentStore = getCurrentStore(stores)!;
 
   const [selectedStoreId, setSelectedStoreId] = useState(currentStore?.id);
+  const [selectedCategoryId, setSelectedCategoryId] = useState<string>();
   const detachedLoadoutMenu = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
+    if ($featureFlags.mobileCategoryStrip) {
+      setSelectedCategoryId('Weapons');
```

Comment:
I think you can just initialize the state conditionally (in `useState`) e.g. `useState($featureFlags.mobileCategoryStrip ? 'Weapons' : undefined)`